CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
-## [development]
+## [3.35.0]
```

Comment:
There should be a space here 👀

Reply:
Where? Above of `## [3.35.0]`?

Reply:
Yes, above 🙏

Reply:
Thanks! Added in 1b0b0380 (and e5a99b98 for the release branch)